plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```
@@ -286,14 +287,14 @@ store< MiniCart >(
 			},
 
 			get badgeBackgroundColor(): string | undefined {
-				if ( state.isHydrated ) {
+				if ( state.isHydrated && ! state.hasBadgeColorAttribute ) {
 					const { ref } = getElement();
 					return getClosestColor( ref!, 'color' ) || '#000';
 				}
 			},
```

Biome (2.4.6)

[error] This getter should return a value.

(lint/suspicious/useGetterReturn)